### PR TITLE
Code Style Fixes done with PHPCS 3.5 

### PR DESCRIPTION
**Pull request by ravage84**
Type: refactoring
Issue: Resolves none
Breaking change: no

This is an alternative to #759 without the PHPCS version that demands PHP 7.1.

### CODE REVIEW

File: src/main/php/PHPMD/Node/ASTNode.php [hunk 1]
```diff
@@ -51,6 +51,7 @@ public function __construct(\PDepend\Source\AST\ASTNode $node, $fileName)
      * @param \PHPMD\Rule $rule
      * @return boolean
      * @SuppressWarnings("PMD.UnusedFormalParameter")
+     * @phpcsSuppress SlevomatCodingStandard.Functions.UnusedParameter
      */
     public function hasSuppressWarningsAnnotationFor(Rule $rule)
     {
```
Comment (kylekatarnls) [inline src/main/php/PHPMD/Node/ASTNode.php:57]: `unset($rule)` as suggested by @tvbeek would be fine I guess and would also allow to remove `@SuppressWarnings`

**Comment by ravage84** [comment]
This is an alternative to #759 without the PHPCS version that demands PHP 7.1.

File: src/main/php/PHPMD/Node/ASTNode.php (hunk 1 above)
Comment (ravage84) [reply src/main/php/PHPMD/Node/ASTNode.php:57]: In this PR, this is simply not necessary, as there is no such PHPCS sniff. I'll remove it.

**Comment by ravage84** [comment]
I withdraw this PR in favor of #759

**Pull request closed without merging**